### ISSUE TITLE
Model stopped learning after some epochs

### ISSUE THREAD

**Issue by NilakshanKunananthaseelan** [flairNLP/flair#2147]
Hi, I am training a NER dataset with IOB format

```
standard O
care O
or O
resection B-group2
and I-group2
standard I-group2
care I-group2
alone I-group2
. O
```

My training script as follows:

```

from flair.data import Corpus
from flair.datasets import ColumnCorpus

train_file = 'rct_hunflair_train_iob.txt'
eval_file = 'rct_hunflair_eval_iob.txt'
test_file = 'rct_hunflair_test_iobes.txt'
 
 
columns = {0:'token',1:'ner'}
 
data_folder = '.' 
corpus: Corpus = ColumnCorpus(data_folder, columns,
                              train_file = train_file,
                              test_file = eval_file,
                              )

corpus = corpus.downsample(0.5)
stats = corpus.obtain_statistics()
print(stats)

from flair.embeddings import WordEmbeddings, FlairEmbeddings, StackedEmbeddings,BertEmbeddings,BytePairEmbeddings,TransformerWordEmbeddings

embedding_types = [

    
    WordEmbeddings("pubmed"),

     
    FlairEmbeddings("pubmed-forward"),
    FlairEmbeddings("pubmed-backward"),
    FlairEmbeddings("pubmed-2015-forward"),
    FlairEmbeddings("pubmed-2015-backward"),
    
]
 
tag_dictionary = corpus.make_tag_dictionary(tag_type="ner")
embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)

from flair.models import SequenceTagger

tagger: SequenceTagger = SequenceTagger(
    hidden_size=256,
    embeddings=embeddings,
    tag_dictionary=tag_dictionary,
    tag_type="ner",
    use_crf=True,
    train_initial_hidden_state=True,
    word_dropout= 0.,
    locked_dropout= 0.,
)

from flair.trainers import ModelTrainer


trainer: ModelTrainer = ModelTrainer(tagger, corpus)

trainer.train(
     base_path="/content/drive/MyDrive/Colab Notebooks/run1",
    max_epochs=350,
    learning_rate=0.1,
    mini_batch_size=20,
    anneal_factor=0.95,
    patience=3,
    monitor_train=True,
    monitor_test=True,
    min_learning_rate= 0.0000001,
   
    weight_decay=1e-6,
    
)

```

![download](https://user-images.githubusercontent.com/25349618/110811376-a76f3100-82ac-11eb-9965-4dd0d5a6860c.png)
It doesn't learn after some epochs,the DEV and TEST F1 score remains around 0.28 to 0.35 .I have tried to reduce the annel_factor, but experiencing same results

**Comment by alanakbik** [comment]
@NilakshanKunananthaseelan could you try removing the weight decay?

**Comment by NilakshanKunananthaseelan** [comment]
![Screenshot_20210314-092053_Slack.jpg](https://user-images.githubusercontent.com/25349618/111056640-a0137780-84a6-11eb-9016-921b6e02c238.jpg)
I tried without weight decay.Does this graph means my test data contains hard samples?

**Comment by NilakshanKunananthaseelan** [comment]
I tried removing weight decay, it improves a bit only? What dropout and locked_dropout do?
